app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor i not vaccinated")

```

Comment:
Code Style: Your error message is not descriptive enough. Try to provide more information about the error. For example, you could say 'The visitor is not vaccinated. Vaccination is required to enter the cafe.' instead of 'Visitor i not vaccinated'.